src/dataflow/src/server/metrics.rs

```diff
@@ -226,5 +234,10 @@ impl CommandsProcessedMetrics {
             self.shutdown.inc_by(self.shutdown_int as i64);
             self.shutdown_int = 0;
         }
+        if self.advance_all_local_inputs_int > 0 {
+            self.shutdown
```

Comment:
Is it intentional that this is incrementing `self.shutdown`? if so I think this merits a comment explaining why?

Reply:
Oops, fixed!